Source/concepts/ConceptAs.ts

```diff
@@ -125,3 +129,10 @@ export class ConceptAs<T extends ConceptBase, U extends string> implements IEqua
         return this.value.toString();
     }
 }
+
+export function conceptFrom<C extends ConceptAs<T, U>, T extends ConceptBase, U extends string>(conceptConstructor: Constructor<ConceptAs<T, U>>, concept: T): C {
+    return new conceptConstructor(concept) as C;
+}
+export function conceptFromFor<C extends ConceptAs<T, U>, T extends ConceptBase, U extends string>(conceptConstructor: Constructor<ConceptAs<T, U>>): (concept: T) => C {
+    return (concept: T) => conceptFrom(conceptConstructor, concept) as C;
+}
```

Comment:
I had a fat comment here about how to use this extra fancy `conceptFromFor()()` thingy but I accidentally closed the tab so in short:

Is there a reason you'd ever want to use the `()()` fancyness over the clear `conceptFrom(SomeId, id)` method?

Reply:
Like you can add a static method like this to your concept class:
```typescript
class SomeId ...{
    /// ... class stuff ...

    static from: (concept: SomeId) {
        return conceptFromFor(SomeId);
    }
}

const someId = SomeId.from(Guid.create());
```

Or you could make reference on spot:
```typescript
const toSomeId = conceptFromFor(SomeId);
const someId = toSomeId(Guid.create());
```

Reply:
Yes, this was the scenarios I had in mind. Since we don't have a way to inherit this static method in a good way